inoculate/src/lib.rs

```diff
@@ -90,19 +90,23 @@ impl Options {
             .suggestion("have you tried not having it be missing")
     }
 
+    pub fn wheres_the_kernel_bin(&self) -> Result<PathBuf> {
+        tracing::debug!("where's the kernel binary?");
+        self.kernel_bin
+            // the bootloader crate's build script gets mad if this is a
+            // relative pathe
```

Comment:
typo (yes, it's my typo and you just copied and pasted it, lol)
```suggestion
            // relative path
```